components/Dashboard/Dashboard.js

```diff
@@ -18,15 +25,99 @@ export default function Dashboard() {
     setSelectedColumn(column);
   }
 
+  const { allEntries, isLoadingEntries, errorEntries } =
+    useData().fetchedAllEntries;
+  const { userEntries } = useData().fetchedUserEntries;
+  const { data: session } = useSession();
+
+  const last7DaysEntries =
+    !isLoadingEntries && last7DaysAnalyser(session ? userEntries : allEntries);
+
+  const allEmotionsDisplayed = entriesToDisplay({
+    last7DaysEntries,
+    session,
+    isLast7Days,
```

Comment:
you pass this element to your function, but it is not destructured there.

Reply:
Which element do you specifically mean?

Reply:
isLast7Days